src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml

```diff
@@ -1528,20 +1549,6 @@ services:
         tags:
             - { name: form.type }
 
-    form.type.sell.product.options.references_type:
```

Comment:
BC Break : services existing in 178x

Reply:
if it's introduced in non released version, and then removed, why it's BC Break?

Reply:
nevermind, it will be available in 1.7.8, and then removed in 8.0.0 👍

Reply:
Besides it's just a form type definition, form types are never accessed directly via the container And this one is part of the experimental page so we are free to break/modify anything we want

But I'll add this into the long list of things I already broke if it's important for you ^^